blog/2024.8.20-Teaming.md

```diff
@@ -15,48 +15,28 @@ tags: [公告]
 
 :::
 
-
 1. OFFC参赛pp范围变更为<FntColor color="#198dff">4500-6800</FntColor>（无偏差值），<FntColor color="#198dff">etx≤6.4</FntColor>
-
-2. 每队八人，一共八队，组队阶段为8.20-9.6
-
-<!-- truncate -->
-
+2. 每队八人，一共八队，组队阶段为8.20-9.6<!-- truncate -->
 3. 取报名成功时间前64位选手进入确认参赛区，其余选手进入待定区作为替补
-
 4. 组队采用cost制，具体cost见主表格，公式为：
-
-<br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![formula](../static/img/teamup_formula.svg)
-
+<br /><div class="align-center"><img src="/img/teamup_formula.svg" class="auto-invert" alt="Cost formula" /></div>
 5. 每队前四<FntColor color="#198dff">cost之和≤24.6</FntColor>
-
 6. 每队<FntColor color="#198dff">etx≥6.0</FntColor>的选手**至多三名**
-
 7. 当队伍人数大于等于4后，队长可私聊OFFC工作人员申报队伍，cost和etx以主表格为准（精确到小数点后两位），最终cost和etx按报名结束时计算，若报名结束时超出组队限制，则队伍需要重新申报
-
 8. 所有报名选手必须更改群名片使之包含你的osu!用户名，每队队长请加入队长QQ群 **984168382**
 
 ---
 
 <details>
   <summary>发布于**2024.8.20**的旧版公告</summary>
-  1. OFFC参赛pp范围变更为<FntColor color="#198dff">4500-6800</FntColor>（无偏差值），<FntColor color="#198dff">etx≤6.4</FntColor>
 
+  1. OFFC参赛pp范围变更为<FntColor color="#198dff">4500-6800</FntColor>（无偏差值），<FntColor color="#198dff">etx≤6.4</FntColor>
   2. 每队七人，一共八队，组队阶段为8.20-9.6
-  
   3. 取报名成功时间前56位选手进入确认参赛区，其余选手进入待定区作为替补；若某一时刻报名成功人数≥64，则取报名成功时间前64位选手进入确认参赛区，其余选手进入待定区作为替补，此时变为每队八人，队伍需要重新申报
-  
-  4. 组队采用cost制，具体cost见主表格，公式为：
-  
-  <br />
-  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![formula](../static/img/teamup_formula.svg)
-
+  4. 组队采用cost制，具体cost见主表格，公式为：<br /><div class="align-center"><img src="/img/teamup_formula.svg" class="auto-invert" alt="Cost formula" /></div>
```

Comment:
Thanks for this *elegant* solution, as I was in a rush to push this document out